molecularnodes/io/local.py

```diff
@@ -75,14 +74,18 @@ def execute(self, context):
         if not scene.MN_import_node_setup:
             style = None
 
+        if not scene.MN_import_centre:
```

Comment:
`MN_import_centre` is a string type, which Blender won't let be `None`, so we don't have to check if it exists, instead we can just say 
```py
centre = scene.MN_centre_type
```

Reply:
Done.